app/models/comment.rb

```diff
@@ -34,12 +34,13 @@ def self.counts_for(commentables)
   # @return [Hash] Hash of counts indexed by request_id
   #
   def self.counts_for_requests(requests) # rubocop:todo Metrics/AbcSize
-    all_commentables = requests.flat_map { |request| [request, request.asset, request.asset&.labware] }
+    all_commentables = requests.flat_map { |request| [request, request.try(:asset), request.try(:asset).try(:labware)] }
```

Comment:
Why use `.try` instead of `&.`?  `&.` protects against the object to its left being `nil` and short circuits to give a `nil` value for the whole method chain.  `.try` attempts to call a method on the object and protects against that method not being implemented.  I think you want to be using `&.` here.  I also assume that having `nil`s all over isn't going to cause issues downstream?